pkg/kubectl/resource/builder.go

```diff
@@ -360,13 +396,33 @@ func hasCombinedTypeArgs(args []string) (bool, error) {
 	switch {
 	case hasSlash > 0 && hasSlash == len(args):
 		return true, nil
-	case hasSlash > 0 && hasSlash != len(args):
-		return true, fmt.Errorf("when passing arguments in resource/name form, all arguments must include the resource")
```

Comment:
It seems if user passes "resource/name1, name2, name3", name2 and name3 will be ignored silently. 

I may be wrong, the code is too complex :)